docs/src/testing/coverage.md

```diff
@@ -40,3 +42,17 @@ If you want `snforge` to call `cairo-coverage` on generated files automatically,
 ```shell
 $ snforge test --coverage
 ```
+
+This will generate a coverage report in the `coverage` directory named `coverage.lcov`.
+
+## Coverage report
```

Comment:
I am not sure that this section is crucial in the forge documentation. However, if we want to include it here I suggest to modify wording a bit. As it is currently written, it seems to imply generating a coverage report from another coverage report, which may be confusing for readers.